chromium_src/chrome/browser/ui/views/tabs/tab_group_style.cc

```diff
@@ -20,6 +20,16 @@
 
 const int TabGroupStyle::kStrokeThicknessForVerticalTabs = 4;
 
+bool TabGroupStyle::TabGroupUnderlineShouldBeHidden() const {
```

Comment:
please include an explanation of why we are overriding the behavior

Reply:
cc: @zenparsing

Reply:
"Upstream currently hides the tab group underline in certain scenarios, whereas we always show the underline."

This is a refresh2023-related fix.

Reply:
Comment added in 29c1410c35ed1b297e4dad8289bb21789b3ed472